<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="leave_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
   <widget class="QLabel" name="build_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>digitalk</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1011</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="side_menu">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/icons/home1.png</normaloff>:/icons/home1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/icons/home-security.png</normaloff>:/icons/home-security.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stack_widget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page_3">
        <widget class="info_widget" name="bedroom1_info" native="true">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>220</y>
           <width>260</width>
           <height>160</height>
          </rect>
         </property>
        </widget>
        <widget class="info_widget" name="wc_info" native="true">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>10</y>
           <width>260</width>
           <height>160</height>
          </rect>
         </property>
        </widget>
        <widget class="info_widget" name="livingroom_info" native="true">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>260</width>
           <height>160</height>
          </rect>
         </property>
        </widget>
        <widget class="info_widget" name="kitchen_info" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>260</width>
           <height>160</height>
          </rect>
         </property>
        </widget>
        <widget class="info_widget" name="bedroom2_info" native="true">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>220</y>
           <width>260</width>
           <height>160</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>171</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Music</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>221</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>Security</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QVideoWidget" name="video_widget" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>585</width>
           <height>480</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>750</x>
           <y>10</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="text">
          <string>REC</string>
         </property>
        </widget>
        <widget class="QPushButton" name="take_photo_button">
         <property name="geometry">
          <rect>
           <x>750</x>
           <y>110</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="text">
          <string>Photo</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>151</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>info_widget</class>
   <extends>QWidget</extends>
   <header>headers/info_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
